netstandard/ref/System.Reflection.Emit.cs

```diff
@@ -10,13 +10,21 @@ public sealed partial class AssemblyBuilder : System.Reflection.Assembly
         public override string FullName { get { throw null; } }
         public override bool IsDynamic { get { throw null; } }
         public override System.Reflection.Module ManifestModule { get { throw null; } }
+        public void AddResourceFile(string name, string fileName) { }
+        public void AddResourceFile(string name, string fileName, System.Reflection.ResourceAttributes attribute) { }
         public static System.Reflection.Emit.AssemblyBuilder DefineDynamicAssembly(System.Reflection.AssemblyName name, System.Reflection.Emit.AssemblyBuilderAccess access) { throw null; }
         public static System.Reflection.Emit.AssemblyBuilder DefineDynamicAssembly(System.Reflection.AssemblyName name, System.Reflection.Emit.AssemblyBuilderAccess access, System.Collections.Generic.IEnumerable<System.Reflection.Emit.CustomAttributeBuilder> assemblyAttributes) { throw null; }
         public System.Reflection.Emit.ModuleBuilder DefineDynamicModule(string name) { throw null; }
 //      Excluded because persistence of Ref Emit is only supported in .NET Framework
 //      public System.Reflection.Emit.ModuleBuilder DefineDynamicModule(string name, bool emitSymbolInfo) { throw null; }
 //      public System.Reflection.Emit.ModuleBuilder DefineDynamicModule(string name, string fileName) { throw null; }
 //      public System.Reflection.Emit.ModuleBuilder DefineDynamicModule(string name, string fileName, bool emitSymbolInfo) { throw null; }
+        public System.Resources.IResourceWriter DefineResource(string name, string description, string fileName) { throw null; }
```

Comment:
The APIs to emit managed resources do not make a lot of sense without persistence.

Reply:
Also excluded